src/components/SiteSearch.vue

```diff
@@ -124,14 +122,17 @@ export default class SiteSearch extends Vue {
   quickLinks!: any[];
 
   get suggestedLinks() {
-    return this.quickLinks.filter(i => {
-      let findResult: any = this.searchData.find(
-        data => data.name === i.item.name
+    return this.quickLinks.reduce((acc, i) => {
+      let jsonSearchIndex = this.jsonSearch.findIndex(
+        (data) => data.name === i.item.name
       );
-      let suggestResult: any = this.searchData.indexOf(findResult);
-      this.quickLinkLoc.push(suggestResult);
```

Comment:
Mutating inside of a computed method is super side-effecty. Also, if this ever did re-compute, wouldn't the list just grow and grow indefinitely? 

The whole point was to map suggestedLink -> jsonSearchIndex.  So just add that key to the suggestedLinks.

Reply:
Action: replace quickLinkLoc with suggested.jsonSearchIndex everywhere.

Action: For `limitedResult`, there's a `.refIndex` that points to the jsonSearch object. So use that.

Reply:
Didn't notice this on my migration. Nice find!